build.gradle

```diff
@@ -225,6 +223,10 @@ dependencies {
     xjc group: 'org.glassfish.jaxb', name: 'jaxb-runtime', version: '3.0.2'
 }
 
+configurations.implementation {
+    exclude group: 'commons-logging', module: 'commons-logging'
```

Comment:
I tihink you have to simply do  exclude module

Reply:
Just followed this one. https://tomgregory.com/how-to-exclude-gradle-dependencies/
Seems to work now, feel free to tweak

Reply:
I rephrase: seems to compile now. But still errors in database test...

Reply:
Yeah, think we need to debug this locally, but currently no time for this.